seametrics/horizon/horizon.py

```diff
@@ -7,12 +7,15 @@ class HorizonMetrics:
 
     Parameters
     ----------
+    vertical_fov_degrees : float
+        The vertical field of view of the camera in degrees.
+    height : int
+        The height of the camera in pixels.
     roll_threshold : float, optional
         The roll threshold in radians. Defaults to 0.5.
     pitch_threshold : float, optional
         The pitch threshold in radians. Defaults to 0.1.
```

Comment:
Hi! Last doubt, I promise haha.

It is not completely clear what the threshold is for. Maybe we can make it clearer?